install_config/upgrading/manual_upgrades.adoc

```diff
@@ -911,53 +911,46 @@ Elasticsearch while running this and it will not perform an install.
+
+- The update process stops all the metrics containers,
+updates the metrics configuration files,
+and redeploys the newer components.
 
-To update to a newer version of the metrics components you will need to use the `refresh`
-mode of the deployer. This will stop all the metrics containers, update the metrics configuration
-files and redeploy the newer components. The `refresh` option will not change the metrics route or
-delete the metrics persistent volume claim.
+- It does not change the metrics route.
 
-Metrics stored to persistent volumes will still contain the persisted data when the updated
-metrics are brought back online. 
+- It does not delete the metrics persistent volume claim.
+Metrics stored to persistent volumes before the update
+are available after the update completes.
 
 [IMPORTANT]
 ====
-All non-persisted metric values will be lost during the metric `refresh`.
-
-All local changes to the metrics configurations will be overwritten with new configurations meant for the latest install.
-This includes things like number of instances in a replica set.
+The update deletes all non-persisted metric values
+and overwrites local changes to the metrics configurations.
+For example, the number of instances in a replica set is not saved.
 ====
 
-To perform the update you will need to follow the
-link:../../install_config/cluster_metrics.html#deploying-the-metrics-components[same steps]
-as when the metrics components were first deployed except this time you will need to specify the
-`MODE=refresh` option.
-
-You will also need to make sure you are using the correct template for your installation. Please see the following
-link:../../install_config/cluster_metrics.html#modifying-the-deployer-template[section] for the location of this template.
+To update, follow the same steps as when the metrics components were
+link:../../install_config/cluster_metrics.html#deploying-the-metrics-components[first deployed],
+using the
+link:../../install_config/cluster_metrics.html#modifying-the-deployer-template[correct template],
+except this time, specify the `MODE=refresh` option:
 
-For instance, if before you deployed metrics using:
-
-----
-$ oc new-app -f metrics-deployer.yaml \
-    -p HAWKULAR_METRICS_HOSTNAME=hawkular-metrics.example.com
-----
-
-You will now need to run:
+====
 ----
 $ oc new-app -f metrics-deployer.yaml \
-    -p HAWKULAR_METRICS_HOSTNAME=hawkular-metrics.example.com,MODE=redeploy
+    -p HAWKULAR_METRICS_HOSTNAME=hm.example.com,MODE=refresh <1>
 ----
-
-After this the metrics components should have been restarted and are now using the most recent versions.
-
+<1> In the original deployment command, there was no `MODE=refresh`.
+====
 
 [NOTE]
 ====
-It is normal to see a gap in the graphs during the time when the `refresh` is occuring. This is due to the
-metrics components not running and being available to gather metrics during this time.
+It is normal to see a gap in the graphs during the `refresh` operation.
```

Comment:
"It is" is expletive. 

http://writing.wisc.edu/Handbook/CCS_expletive.html

Maybe:

"Seeing gaps in the graphs during the `refresh` operation is normal."